cmake/external/abseil-cpp.cmake

```diff
@@ -21,7 +21,7 @@ FetchContent_Declare(
     abseil_cpp
     PREFIX "${CMAKE_CURRENT_BINARY_DIR}/abseil-cpp"
     BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/external/abseil-cpp"
-    URL https://github.com/abseil/abseil-cpp/archive/refs/tags/20211102.0.zip
+    URL ${DEP_URL_abseil_cpp}
```

Comment:
> URL ${DEP_URL_abseil_cpp} [](http://example.com/codeflow?start=3&length=26)

The problem with Abseil, is that its internal structures can be version depended and that breaks Natvis.

Reply:
This PR does not update abseil. It just moves such URLs to a central place: deps.txt

Reply:
However, it does allow the override.

Reply:
So your suggestion is?

Reply:
Natvis allows versioned visualization, this means when updating a version, the person must add new visualisation, but the URL is now disjoint with this cmake file where the comment states that. So we may need to replicate that comment in the main make file, unless we move those `Fetches` in a separate cmake file.

Reply:
Got it. I added a comment in deps.txt.